app/scripts/early_voting_mgr.js

```diff
@@ -25,7 +25,15 @@ define(
     function whenMarkerEventsHappen(eventType, marker) {
       if (eventType === 'click') {
         for (var i = 0; i < earlyVotingLocations.length; i++) {
+
           if (marker.getPosition().equals(earlyVotingLocations[i].getGeometry().get())) {
+            
+            var collapsedToggle = Boolean(document.getElementById('#collapse-toggle'+i).getAttribute("aria-expanded"));
+
+            if (collapsedToggle) {
+              $('#collapse'+i).collapse('show');       
```

Comment:
with some of the work that @WheresHJ did on the design, I'm not sure if we want to do this (as some of the most important information will be displayed in the collapsed state) - I'm going to cherry-pick the commit for the address change to pull that in.